data/mods/Magiclysm/recipes/alchemy.json

```diff
@@ -69,5 +69,33 @@
     "category": "CC_ENCHANTED",
     "subcategory": "CSC_ENCHANTED_OTHER",
     "autolearn": true
+  },
+  {
+    "type": "recipe",
+    "activity_level": "fake",
+    "result": "enchanted_wood",
+    "qualities": [ { "id": "MANA_INFUSE", "level": 1 } ],
+    "components": [ [ [ "mana_dust", 10 ] ], [ [ "stick_long", 1 ] ] ],
+    "time": "30 m",
+    "skill_used": "spellcraft",
+    "difficulty": 4,
+    "proficiencies": [ { "proficiency": "prof_alchemy" } ],
+    "category": "CC_ENCHANTED",
+    "subcategory": "CSC_ENCHANTED_OTHER",
+    "autolearn": true
+  },
+  {
+    "type": "recipe",
+    "activity_level": "fake",
+    "result": "mana_pearl",
```

Comment:
This would appear to be much easier to get/make than a bulette pearl, and not much more difficult than the enchanted wood that is already required. Is this appropriate?

Reply:
Mana pearl is an ingredient for the lesser staff while bulette pearl is for the greater version of the staff. So it's make sense that it's easier to get.